src/Uno.UI/UI/Xaml/Controls/TextBox/TextBoxView.skia.cs

```diff
@@ -45,7 +45,7 @@ public TextBoxView(TextBox textBox)
 
 		internal void SetTextNative(string text)
 		{
-			DisplayBlock.Text = text;
+			DisplayBlock.Text = new string('•', text.Length);
```

Comment:
The problem should probably be here. I'll take a look to confirm and fix. Sorry for introducing such a regression :(

Reply:
No worries, happens to everybody. :)

For my part, I somehow did not even notice this line when I reviewed the PR.